app/Http/Controllers/DepartmentController.php

```
@@ -0,0 +1,74 @@
+use App\Models\Department;
+use Illuminate\Http\Request;
+use Illuminate\Support\MessageBag;
+
+class DepartmentController extends Controller
+{
+    public function create()
+    {
+        return view('organisation.departments.edit');
+    }
+
+    public function edit($departmentId)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$departmentId'. (Unused Code Rules)

(UnusedFormalParameter)